scriptmodules/emulators/mupen64plus/mupen64plus.sh

```diff
@@ -231,7 +231,11 @@ function testCompatibility() {
             fi
             iniConfig " = " "" "$config"
             # Settings version. Don't touch it.
-            iniSet "configVersion" "17"
+            local config_version="17"
+            if [[ -f $configdir/n64/GLideN64_config_version.ini ]]; then
```

Comment:
my apologies - I misread (was doing the review on my mobile ;-) )

Please can you still quote it and squash and I will merge.